python/cudf_polars/cudf_polars/containers/scalar.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+"""A scalar, with some properties."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import cudf._lib.pylibcudf as plc
+
+__all__: list[str] = ["Scalar"]
+
+
+class Scalar:
```

Comment:
Minor thing, especially since things could still change, but right now this really seems like a dataclass.